scss/composer/_emoji-sticker-picker.scss

```diff
@@ -32,6 +33,10 @@ $emojimart-search-height: 31px + 6px;
   height: calc(#{$esp-header-height} + 1px);
   display: flex;
   border-bottom: 1px solid var(--emojiMartBorder);
+  background: var(--emojiMartBg);
+  border-top-left-radius: var(--emojiMartOutsideRadius);
+  border-top-right-radius: var(--emojiMartOutsideRadius);
+
 }
 
```

Comment:
This Made the a merge conflict. delete neither and just put this in between

Reply:
try to rebase your branch to master